python/lsst/pipe/base/cmdLineTask.py

```diff
@@ -50,7 +50,7 @@ def _poolFunctionWrapper(function, arg):
         return function(arg)
     except Exception:
         raise  # No worries
-    except:
+    except:  # noqa E722
```

Comment:
See the discussion in lsst/pipe_supertask#17 where I discussed this exact problem with @andy-slac -- I think this should be `BaseException` or should maybe disappear completely in Python 3 land.